buddyboss/buddyboss-platform
# Fixed icons issue on text formatting toolbar

### All Submissions:


Fixes #1051 #726 .

### How to test the changes in this Pull Request:

1. Install and activate BuddyPress User Blog
2. Go to Messages on frontend
3. Click on Show Formatting
4. See error
5. Go to Forums
6. Click on Reply
7. Click on the font formatting icon
8. See error

### Proof Screenshots or Video
https://prnt.sc/t266ey
https://prnt.sc/t266z2

### Other information:
